.github/configs/ct-install.yaml

```diff
@@ -16,3 +16,5 @@ validate-maintainers: true
 validate-yaml: true
 exclude-deprecated: true
 excluded-charts: []
+# debug applicationset
+namespace: argocd
```

Comment:
this allows ci to pass because application set needs to be in same namespace than argo